composer.json

```diff
@@ -240,7 +240,7 @@
         "ratchet/pawl": ">=0.4.1",
         "react/http": ">=1.6.0",
         "react/async": "^4.0.0",
-        "react/promise": "^2.9.0",
+        "react/promise": ">=3.0.0",
```

Comment:
Why not this?

```suggestion
        "react/promise": "^3.0.0",
```

>= is unbounded and dangerous

Reply:
right, i've had `^` initially, just PR build didn't install v3 and I've just switched to that to test. however, switched back now as you pointed out.